data/ExperimentDesignPresets.ts

```diff
@@ -0,0 +1,26 @@
+import {EmptyAAExperiment} from "../types/experiments";
+
+interface Preset<T> {
+  name: string;
+  description: string;
+  presets: Partial<T>
+}
+
+export const AAType: Preset<EmptyAAExperiment> = {
```

Comment:
Can you update this to be a default export to match #43? I think when we have more we'll either want to put one preset per file, like `data/experimentDesignPresets/AAType.ts` or have this file export a group like

```typescript
export default { AAType, ABType, ABCType }
```